e2e-tests/IIoTPlatform-E2E-Tests/TestHelper.cs

```diff
@@ -165,132 +183,115 @@ internal static class TestHelper {
         }
 
         /// <summary>
-        /// Switch to publisher orchestrated mode
+        /// Sets the unmanaged-Tag to "true" to enable Orchestrated-Mode
         /// </summary>
-        /// <param name="destinationFilePath">Path of the PublishedNodesFile.json file to be deleted</param>
         /// <param name="context">Shared Context for E2E testing Industrial IoT Platform</param>
-        /// <param name="ct">Cancellation token</param>
-        public static async Task SwitchToOrchestratedModeAsync(
-            string destinationFilePath,
-            IIoTPlatformTestContext context,
-            CancellationToken ct = default
-        ) {
+        /// <returns></returns>
+        public static async Task SwitchToOrchestratedModeAsync(IIoTPlatformTestContext context, CancellationToken ct = default) {
             var patch =
-               @"{
+                @"{
                     tags: {
-                        unmanaged: null
+                        unmanaged: false
```

Comment:
This wont work, the layered deployment condition is `NOT IS_DEFINED(tags.unmanaged)`, that means unmanged need to be set to **null**

Reply:
This explains the failed build. :-)